Rakefile

```
@@ -1,49 +1,51 @@
 # frozen_string_literal: true
 
-# Migrate
+require "sequel"
 
-migrate = lambda do |env, version|
+# Migrate
+migrate = lambda do |env, version, db: nil|
   ENV["RACK_ENV"] = env
   require "bundler/setup"
   Bundler.setup
   require "logger"
-  require "sequel"
   require_relative "db"
   Sequel.extension :migration
-  DB.extension :pg_enum
 
-  DB.loggers << Logger.new($stdout) if DB.loggers.empty?
-  Sequel::Migrator.apply(DB, "migrate", version)
+  db ||= DB
+  db.extension :pg_enum
+
+  db.loggers << Logger.new($stdout) if db.loggers.empty?
+  Sequel::Migrator.apply(db, "migrate", version)
 
   # Check if the alternate-user password hash user needs to run
   # migrations.  It's desirable to avoid always connecting to run
   # migrations, since, almost always, there will be nothing to do and
   # it gluts output.
-  case DB[<<SQL].get
+  case db[<<SQL].get
 SELECT count(*)
 FROM pg_class
 WHERE relnamespace = 'public'::regnamespace AND relname = 'account_password_hashes'
 SQL
   when 0
-    user = DB.get(Sequel.lit("current_user"))
+    user = db.get(Sequel.lit("current_user"))
     ph_user = "#{user}_password"
 
     # NB: this grant/revoke cannot be transaction-isolated, so, in
     # sensitive settings, it would be good to check role access.
-    DB["GRANT CREATE ON SCHEMA public TO ?", ph_user.to_sym].get
-    Sequel.postgres(**DB.opts.merge(user: ph_user)) do |db|
-      db.loggers << Logger.new($stdout) if db.loggers.empty?
-      Sequel::Migrator.run(db, "migrate/ph", table: "schema_migrations_password")
+    db["GRANT CREATE ON SCHEMA public TO ?", ph_user.to_sym].get
+    Sequel.postgres(**db.opts.merge(user: ph_user)) do |d|
```

We have `db` and `d`, which is a bit confusing. Since `d` uses `ph_user`, should we rename it as `ph_db`?
```suggestion
    Sequel.postgres(**db.opts.merge(user: ph_user)) do |ph_db|
```